mixer/tools/codegen/pkg/bootstrapgen/template/bootstrapTemplate.go

```diff
@@ -433,7 +433,7 @@ var (
                     abag = newWrapperAttrBag(
                         func(name string) (value interface{}, found bool) {
                             field := strings.TrimPrefix(name, fullOutName)
-                            if len(field) != len(name) {
+                            if len(field) != len(name) && out.WasSet(field) {
```

Comment:
Can you make the same change to the runtime2 bindings below? Around line 547 or so I believe.